Wrappers/Python/test/test_io.py

```diff
@@ -361,5 +362,83 @@ def TIFF_compression_test(self, compression, channels=1):
         
         np.testing.assert_array_equal(tmp, read_array)
 
+class TestRAW(unittest.TestCase):
+    def setUp(self) -> None:
+        self.TMP = tempfile.TemporaryDirectory()
```

Comment:
do these all share the same temporary directory?

Reply:
no, each test creates its own